Softeq.XToolkit.WhiteLabel.iOS/AppDelegateBase.cs

```diff
@@ -73,11 +72,23 @@ protected virtual void StartScopeForIoc()
         {
             var containerBuilder = new ContainerBuilder();
             ConfigureIoc(containerBuilder);
+            RegisterServiceLocator(containerBuilder);
             RegisterInternalServices(containerBuilder);
 
             Dependencies.IocContainer.StartScope(containerBuilder);
         }
 
+        protected virtual void RegisterServiceLocator(ContainerBuilder builder)
```

Comment:
I see completely code duplicates, It would be great to move it to platform independent site and just reuse it on Android and iOS. @AlexandraMash what do you think?

Reply:
Yes, I was thinking about the same thing. I was not sure which component can be used for this purpose, I don't think any of the existing ones will suite. And further maybe we could separate the whole flow of setting up Ioc from application and app delegate to some common component cause they are quite similar and besides are partially repeated in AutoRegistration version?